src/libraries/System.Security.Cryptography.Cose/ref/System.Security.Cryptography.Cose.cs

```diff
@@ -55,18 +55,22 @@ public abstract partial class CoseMessage
         public System.ReadOnlyMemory<byte>? Content { get { throw null; } }
         public System.Security.Cryptography.Cose.CoseHeaderMap ProtectedHeaders { get { throw null; } }
         public System.Security.Cryptography.Cose.CoseHeaderMap UnprotectedHeaders { get { throw null; } }
-        public static System.Security.Cryptography.Cose.CoseSign1Message DecodeSign1(byte[] cborPayload) { throw null; }
```

Comment:
We should leave the `byte[]` overload.

Reply:
@AraHaan see https://github.com/dotnet/runtime/pull/66741#discussion_r828546887
> the implicit conversion turns (byte[])null into ReadOnlySpan<byte>.Empty. Often we want the byte[]-accepting overload to be null-rejecting, to provide better a better debugging experience than seeing the empty span.